marathon_acme/tests/fake_marathon.py

```diff
@@ -108,12 +108,14 @@ def get_events(self, request):
         def callback(event):
             _write_request_event(request, event)
             self.client.flush()
+        remote_address = request.getClientAddress().host
```

Comment:
Technically, not every thing that `getClientAddress` returns will have a `host` attribute (e.g. if a Unix socket is used). But we don't really care about that and this is just in tests.